client/landing/stepper/declarative-flow/internals/steps-repository/store-address/index.tsx

```diff
@@ -0,0 +1,256 @@
+import { StepContainer } from '@automattic/onboarding';
+import styled from '@emotion/styled';
+import { ComboboxControl } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import { FormEvent, ReactElement, useState } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormInputValidation from 'calypso/components/forms/form-input-validation';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormInput from 'calypso/components/forms/form-text-input';
+import { useSite } from 'calypso/landing/stepper/hooks/use-site';
+import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ActionSection, StyledNextButton } from 'calypso/signup/steps/woocommerce-install';
+import { useCountries } from '../../../../hooks/use-countries';
+import type { Step } from '../../types';
+import './style.scss';
+
+type FormFields =
+	| 'store_address_1'
+	| 'store_address_2'
+	| 'store_city'
+	| 'store_postcode'
+	| 'store_country';
+
+const CityZipRow = styled.div`
+	display: -ms-grid;
+	display: grid;
+	width: 100%;
+	-ms-grid-columns: 48% 4% 48%;
+	grid-template-columns: 48% 48%;
+	grid-column-gap: 4%;
+	justify-items: stretch;
+`;
+
+const StoreAddress: Step = function StartingPointStep( { navigation } ) {
```

Comment:
The function name here is from the "base" step :)

Reply:
Oops, you're correct! Thanks